src/main/java/com/bootcamp/demo/restapi/ScooterController.java

```diff
@@ -0,0 +1,49 @@
+package com.bootcamp.demo.restapi;
+
+import com.bootcamp.demo.model.Scooter;
+import com.bootcamp.demo.model.component.Location;
+import com.bootcamp.demo.service.ScooterService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping(path = "/api/scooters")
+public class ScooterController {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScooterController.class);
+    private final ScooterService scooterService;
+
+    public ScooterController(ScooterService scooterService) {
+        this.scooterService = scooterService;
+    }
+
+    @GetMapping("/{scooterId}")
+    public ResponseEntity<Scooter> getScooterById(@PathVariable(value = "scooterId") String id) {
+        LOGGER.info("GET SCOOTER BY ID - API endpoint invoked");
+        return scooterService.findScooterById(id);
+    }
+
+    @GetMapping
+    public Set<Scooter> getScooters() {
+        LOGGER.info("GET SCOOTERS - API endpoint invoked");
+        return scooterService.findAllScooters();
+    }
+
+    @PostMapping
+    @ResponseStatus(value = HttpStatus.CREATED)
+    public void postScooter(@RequestBody final Scooter scooter) {
+        LOGGER.info("POST SCOOTER - API endpoint invoked");
+        scooterService.createScooter(scooter);
+    }
+
+    @PostMapping("/available/{searchRadius}")
+    @ResponseStatus(value = HttpStatus.OK)
+    public Set<Scooter> getAvailableScooters(@PathVariable(value = "searchRadius") Double searchRadius, @RequestBody final Location selectedLocation) {
+        LOGGER.info("POST SCOOTER - API endpoint invoked");
```

Comment:
These types of logs are redundant.
Often logging too much at info level results in logs pollution. Logging should be used as long as it is helpful but it is not supposed to be a "finer" form of debugging.
Applications can have several types of logging files i.e. access logs/server logs and the purpose for each is different.
An access log will keep track of all invoked endpoints, whereas in the server log you usually add logs to pick up when something does not work as expected (exceptional/error flows etc.).
You can find a basic intro here: https://www.baeldung.com/spring-boot-embedded-tomcat-logs